src/main/java/org/broadinstitute/consent/http/db/mapper/DataAccessRequestMapper.java

```diff
@@ -27,6 +28,8 @@ public DataAccessRequest map(ResultSet resultSet, StatementContext statementCont
         dar.setSortDate(resultSet.getTimestamp("sort_date"));
         dar.setSubmissionDate(resultSet.getTimestamp("submission_date"));
         dar.setUpdateDate(resultSet.getTimestamp("update_date"));
+        dar.setCountCollaborators(resultSet.getArray("internalCollaborators"));
+        dar.setCountCollaborators(resultSet.getArray("labCollaborators"));
```

Comment:
This class loops over the results of a query and tries to extract the query columns from the results array. In these lines, you're expecting that query results will contain columns for `internalCollaborators` and `labCollaborators`, but those values don't exist in the database as columns on the data access request table, which is the root of the failures. These values have to be extracted from the `data` column but this is probably not the right place to look for those values.

A second bit of feedback here - you're calling the same method (`setCountCollaborators`) with two different things, so if this did work, it would be overwriting the first with the second.